.github/workflows/pypi-release.yaml

```diff
@@ -74,9 +74,8 @@ jobs:
       - name: Check uploaded package
         if: github.event_name == 'push'
         run: |
-          sleep 3
           python -m pip install --upgrade pip
-          python -m pip install --extra-index-url https://test.pypi.org/simple --upgrade xarray
+          python -m pip install dist/xarray*.whl
           python -m xarray.util.print_versions
```

Comment:
Now that it doesn't depend on it, can we move this before the "upload to testpypi" step? That way we can avoid uploading files that are broken (even though for testpypi that's not much of an issue)

Reply:
Done!